test/integration_test.go

```diff
@@ -684,6 +684,20 @@ func (ts *IntegrationTestSuite) TestContextPropagator() {
 	}, propagatedValues)
 }
 
+const CronWorkflowID = "test-cron"
+
+func (ts *IntegrationTestSuite) TestCron() {
```

Comment:
This sounds like we test cron scheduler here but we don't, right? `TestFailurePropogation` probably a better name here. Workflow itself might have a better name.

Reply:
Indeed